NitroxServer/ConsoleCommands/UpgradeCommand.cs

```diff
@@ -34,6 +34,13 @@ protected override void Execute(CallArgs args)
 
             SaveFileVersion saveFileVersion = worldPersistence.Serializer.Deserialize<SaveFileVersion>(Path.Combine(saveDir, $"Version{fileEnding}"));
 
+            // SaveFileVersion structure was updated in V1.5.0.0
+            // This can be removed with V1.6.0.0 or later
```

Comment:
Is this something worth keeping ? I do think it's not worth it maintaining save files at this stage of the developement, imo upgrades would fit better for future version, once the core mod is stabilized

Reply:
I would leave it in for this version. It's just a couple lines of extra code and the upgrade script is already written. On top of that we could use it as a small testing/beta phase on the users as only a few of them were using JSON as save-file-format.